docs/src/testers/sessions/tailnet_proxy.py

```
@@ -0,0 +1,18 @@
+# @sniptest filename=tailnet_proxy.py
+# @sniptest typecheck_only=true
+from notte_sdk import NotteClient
+from notte_sdk.types import ExternalProxy, NotteProxy, TailnetProxy
+
+client = NotteClient()
+
+# Configure a Tailscale tsnet proxy using OAuth client credentials
+tailnet_proxy = TailnetProxy(
+    oauth_client_id="your-tailscale-oauth-client-id",
+    oauth_client_secret="your-tailscale-oauth-client-secret",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "oauth_client_secret"

(S106)